1907-05-27.xml

```diff
@@ -1532,15 +1537,4661 @@
+                    <p>J.E. 950 have been spent since since the first of January on the improvement
+                        of the streets of Khartoum. 16,810 sq. meters of the surface of the streets
+                        have been metalled and, rendered solid in anticipation of the rainy reason.
+                        About L.E. 1700 more are to be expended in 1907 on street improvement.</p>
+                </div>
+                <div type="subsection">
+                    <p><hi rend="bold">Cirder Tenders.</hi></p>
+                    <p>. The Egyptian Railway Administration recently asked for tenders for 25,000
+                        short pine wood girders. Three tenders were received and that of Messrs. Th.
+                        N. Kindineco and Co. was accepted. That firm agreed to supply the girders at
+                        the rate of one hundred and forty milliemes per girder for those which will
+                        have to support car transport and at the rate of one hundred and forty five
+                        milliemes per girder for those which will have to support cart transport</p>
+                </div>
+                <div type="subsection">
+                    <p><hi rend="bold">St. Joseph's College.</hi></p>
+                    <p>Yesterday afternoon a most successful gymnastic and musical drill display was
+                        given by the pupils of Saint Joseph's College, Bacos, under the presidence
+                        of the French Vice Consul, M. Chatles Drenilhes. Invitations were freely
+                        distributed, and an excellent attendance of parents and friends of the
+                        pupils was the result. The course.and groundswere prettily decorated, and
+                        the smartness and precision with which the various movements were carried
+                        out, to the accompaniment of the St. Catherine’s College band, reflected
+                        great credit upon their instructor, Mr. H. Tookey. </p>
+                </div>
+                <div type="subsection">
+                    <p><hi rend="bold">A Smyrna Carpet Trust.</hi></p>
+                    <p>An American Consular report contains the announcement that an agreement has
+                        been reached between the carpet manufactorers of Asia Minor which will bring
+                        about an increase in the price of Smyrna carpets. A combination has been
+                        formed with a number of wool dyers, known as the Amalgamated Oriental Carpet
+                        Manufacturers, with a capital of about £300,000. A large warehouse is to be
+                        erected in Smyrna, where all the carpets, etc., made by the firms in the
+                        combine will be stored and the existing warehouses belonging to these firms
+                        will be closed.</p>
+                </div>
+                <div type="section">
```

Comment:
some features are missing in your file, like personal and social should be "social". Please check https://dig-eg-gaz.github.io/contents/features/